tsc/process/release.md

```diff
@@ -3,27 +3,23 @@
 The following assumes that the current OpenVDB library version number is 6.0.0 and the new version number is 6.1.0. Adjust for the actual version numbers as appropriate.
 
 - [ ] Open a Jira "Release OpenVDB 6.1.0" ticket with "OpenVDB_6.1.0" as the Fix Version.
-- [ ] Update `CHANGES` and `doc/changes.txt` with release notes.  [_Specifics TBD, pending a review of release note management tools._]
+- [ ] Update `CHANGES` and `doc/changes.txt` with release notes. Ensure doc/doxygen-config has the correct release number.  [_Specifics TBD, pending a review of release note management tools._]
 - [ ] Open a pull request to merge the above changes into `openvdb/master`.  Associate the pull request with the Jira ticket created earlier, and verify that the CI build runs successfully.
 - [ ] Draft a new [GitHub release](https://github.com/AcademySoftwareFoundation/openvdb/releases). Title it "OpenVDB 6.1.0" and tag it as `v6.1.0`.
 
 - [ ] Update `openvdb-website/index.html` with a news item announcing the release, and delete the oldest news item.  Open that page in a browser and check that the website renders correctly and that there are no broken links.
-- [ ] Build the documentation (both the `doc` and `pydoc` targets) and replace the contents of `openvdb-website/documentation/doxygen/` with the output. There are two sub folders, /search and /python. /search will be generated by the core doxygen commands. The /python folder will be built by epydoc. The python documentation requires the python module to be built and installed. Make sure you build with OPENVDB_PYTHON_WRAP_ALL_GRID_TYPES=ON, otherwise documentation will not be generated for some grid types. An example CMake command list (assuming within a nested build folder):
+- [ ] Build the documentation (for the core library and python module) using Doxygen 1.8.11 and replace the contents of `openvdb-website/documentation/doxygen/` with the output. There are two sub folders, /search and /python. /search will be generated by the core doxygen commands. The /python folder will be built by epydoc. The python documentation requires the python module to be built and installed. Make sure you build with OPENVDB_PYTHON_WRAP_ALL_GRID_TYPES=ON, otherwise documentation will not be generated for some grid types. An example CMake command list (assuming within a nested build folder):
 
 ```shell
 # Build python module and core documentation
 install_location_on_path=/usr/local
 website_dox_location=openvdb-website/documentation/doxygen/
 cmake ../ \
-    -DCMAKE_INSTALL_PREFIx=$install_location_on_path \
-    -DOPENVDB_BUILD_CORE=ON \
+    -DCMAKE_INSTALL_PREFIX=$install_location_on_path \
     -DOPENVDB_BUILD_PYTHON_MODULE=ON \
     -DOPENVDB_PYTHON_WRAP_ALL_GRID_TYPES=ON \
     -DOPENVDB_BUILD_BINARIES=OFF \
-    -DOPENVDB_BUILD_UNITTESTS=OFF \
     -DOPENVDB_BUILD_DOCS=ON \
-    -OPENVDB_BUILD_HOUDINI_PLUGIN=OFF \
-    -DOPENVDB_BUILD_MAYA_PLUGIN=OFF \
```

Comment:
I had these explicitly to avoid any changing defaults in the future

Reply:
Understood, I opted for simplicity. I don't see us changing these anytime soon.